crates/ruff_python_formatter/tests/snapshots/format@expression__split_empty_brackets.py.snap

```diff
@@ -178,8 +178,8 @@ return await self.http_client().fetch(
     **kwargs,
 )
 
-return (
-    await self()
+return await (
+    self()
```

Comment:
This is actually a deviation from Black now.

Given:

```python
return await self().http_client().fetch(
    f"http://127.0.0.1:{self.port}{path}", method=method, **kwargs,
)
```

We now format as:

```python
return await (
    self()
    .http_client()
    .fetch(
        f"http://127.0.0.1:{self.port}{path}",
        method=method,
        **kwargs,
    )
)
```

Whereas Black formats as:

```python
return (
    await self()
    .http_client()
    .fetch(
        f"http://127.0.0.1:{self.port}{path}",
        method=method,
        **kwargs,
    )
)
```

I need to explore this further.